torchtext/transforms.py

```diff
@@ -657,6 +659,60 @@ def __prepare_scriptable__(self):
         return self
 
 
+class RegexTokenizer(Module):
+    __jit_unused_properties__ = ["is_jitable"]
+    r"""Regex tokenizer for a string sentence that applies all regex replacements defined in patterns_list.
+
+    Args:
+        patterns_list (List[Tuple[str, str]]): a list of tuples (ordered pairs) which contain the regex pattern string
+        as the first element and the replacement string as the second element.
+
+    Examples:
+        >>> import torch
+        >>> from torchtext.transforms import regex_tokenizer
+        >>> test_sample = 'Basic Regex Tokenization for a Line of Text'
+        >>> patterns_list = [
+            (r'\'', ' \'  '),
+            (r'\"', '')]
+        >>> reg_tokenizer = regex_tokenizer(patterns_list)
```

Comment:
Since we've deleted the factory method and will be instantiating the `RegexTokenizer` object directly in code, let's replace `regex_tokenizer` with `RegexTokenizer` in the example docstrinng